package.json

```diff
@@ -7,7 +7,7 @@
   "module": "./dist/index.module.js",
   "exports": {
     "require": "./dist/index.js",
-    "default": "./dist/index.modern.js"
+    "default": "./dist/index.modern.mjs"
```

Comment:
Is this because of the new microbundle version?

Reply:
yes, https://github.com/developit/microbundle/pull/950